examples/sgd/sgd.go

```diff
@@ -9,20 +9,16 @@ func main() {
 	opt := torch.NewSGDOpt(0.1, 0, 0, 0, false)
 	opt.AddParameters([]torch.Tensor{a})
 
+	torch.PrepareGC()
```

Comment:
torch.BeginTraining ?

Reply:
It seems this function 's not necessary.